pyproject.toml

```diff
@@ -0,0 +1,99 @@
+[tool.poetry]
+name = "rero-ebooks"
+version = "0.2.0"
+description = "Invenio digital library framework."
```

Comment:
please change it

Reply:
I changed the version to `0.3.0` and the description to `Ebooks repository for RERO`